src/applications/vic-v2/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "appName": "Veteran ID Card V2",
+  "entryFile": "./veteran-id-card-entry.jsx",
+  "entryName": "vic-v2",
+  "rootUrl": "/veteran-id-card/apply"
+}
```

Comment:
Isn't this one changing to [`/records/get-veteran-id-cards/`](https://docs.google.com/spreadsheets/d/1YucOtYihweNPI1gYtCgdsinKQmH4DMuSgPaSgJMfyYk/edit#gid=316089358)?

Reply:
No, all of the content pages are for VIC v1, which is a different app than this (and isn't a form on our side).

Reply:
This page doesn't exist in the WBC content and I'm not adding an equivalent because VIC v2 is on hiatus for the foreseeable future.

Reply:
Ah gotcha, thank you!